src/app/parser/impl/sol/SolParser.ts

```diff
@@ -0,0 +1,74 @@
+//
+// LESERKRITIKK v2 (aka Reader Critics)
+// Copyright (C) 2017 DB Medialab/Aller Media AS, Oslo, Norway
+// https://github.com/dbmedialab/reader-critics/
+//
+// This program is free software: you can redistribute it and/or modify it under
+// the terms of the GNU General Public License as published by the Free Software
+// Foundation, either version 3 of the License, or (at your option) any later
+// version.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+// FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along with
+// this program. If not, see <http://www.gnu.org/licenses/>.
+//
+
+import * as Cheerio from 'cheerio';
+
+import ArticleAuthor from 'base/ArticleAuthor';
+
+import AbstractLabradorParser from 'app/parser/AbstractLabradorParser';
+import IteratingParserItem from 'app/parser/IteratingParserItem';
+import {cfEmailDecode} from 'app/parser/util/EmailDecode';
+
+export default class SolParser extends AbstractLabradorParser {
+
+	// Implement AbstractParser
+
+	protected parseVersion() : Promise <string> {
+		return Promise.resolve(
+			this.select('div.meta').find('meta[itemprop="dateModified"]').attr('content')
```

Comment:
Comment not for you exactly, but in general

This looks not right, because all Aller sites has normal meta data, but here I see <meta property="article:published_time" content="undefined"> and no modified_time at all

Reply:
yes, I saw the same)